### PR TITLE
Week2 assignment1

### PR DESCRIPTION
**Pull request by relhanakshay**
Pull request for assignment 1 week2

### CODE REVIEW

File: Week-2/index.html [hunk 1]
```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
```
Comment (RajeshSivanesan) [inline Week-2/index.html:1]: @relhanakshay I could see that the content is not responsive. Can you check it out. The articles are not wrapping down the image.

File: Week-2/index.html [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Non corrupti aut cum esse nesciunt itaque, quo
+                quibusdam
+                explicabo vel quia! Vero nam quaerat fuga ad deleniti repudiandae officiis atque ratione voluptatem
+            </article>
+        </div>
+
+    </div>
+
+
+    <form class="form-class">
```
Comment (RajeshSivanesan) [inline Week-2/index.html:67]: @relhanakshay form also seems not responsive. Can you check it out.

File: Week-2/index.html (hunk 1 above)
Comment (RajeshSivanesan) [reply Week-2/index.html:1]: Screenshot 2022-07-30 at 10 57 52 AM

File: Week-2/index.html (hunk 2 above)
Comment (RajeshSivanesan) [reply Week-2/index.html:67]: Screenshot 2022-07-30 at 10 58 50 AM

File: Week-2/index.html (hunk 1 above)
Comment (relhanakshay) [reply Week-2/index.html:1]: Hi Rajesh, I have made it responsive now.

File: Week-2/index.html (hunk 2 above)
Comment (relhanakshay) [reply Week-2/index.html:67]: I have checked that the form is already responsive.

**Comment by relhanakshay** [comment]
Cloned and closing this

**Pull request closed without merging**